migrations/0058_order_history.up.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS order_history (
+    id serial PRIMARY KEY,
+    operation text NOT NULL,
+    executed_by text NOT NULL DEFAULT current_user,
+    recorded_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    order_id uuid NOT NULL,
+    value_before jsonb,
+    value_after jsonb
+);
+
+CREATE OR REPLACE FUNCTION fn_order_insert() RETURNS TRIGGER AS $$
+    BEGIN
+        IF TG_OP = 'INSERT' THEN
```

Comment:
An extra step to make sure this handles only inserts. Similar is done in the other functions.